x-pack/plugins/global_search_providers/public/providers/get_app_results.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import levenshtein from 'js-levenshtein';
+import { PublicAppInfo, PublicLegacyAppInfo } from 'src/core/public';
+import { GlobalSearchProviderResult } from '../../../global_search/public';
+
+export const getAppResults = (
+  term: string,
+  apps: Array<PublicAppInfo | PublicLegacyAppInfo>
+): GlobalSearchProviderResult[] => {
+  return (
+    apps
+      // remove disabled and chromeless app (to avoid returning the login page for example)
+      .filter((app) => app.status === 0 && (app.legacy === true || app.chromeless !== true))
```

Comment:
I'm filtering `chromeless` apps because I don't think it makes any sense to find the `login` or `logout` application. I **think** it's acceptable to declare that our `chromeless` should all be considered as `hidden`.

If it's not the case, we would probably need to introduce an additional field in `App`, as there is currently no way to differenciate chromeless apps that should appear in the results from the one that shouldn't.

Reply:
If we ever need to change our mind in the future, it seems easiest to assume `chromeless` is also `hidden` unless there's also some `showInSearch` field present too which we could add whenever the need comes up.